mle/function/__init__.py

```diff
@@ -247,6 +255,60 @@
             'limit_rows': {
                 'type': 'integer',
                 'description': 'The number of rows to preview, should not be a very large number. Default is 3.'
+            },
+            'limit_columns': {
+                'type': 'integer',
+                'description': 'The number of columns to preview, should not be a very large number. Default is None.'
```

Comment:
type is integer 
why is the `default is None`

Reply:
I think the column contains a lot of feature-related information for data science tasks, so I prefer not to set any limitations on the columns. Still, this parameter provides an approach for the LLM to set it when necessary.